sympy/holonomic/holonomic.py

```diff
@@ -135,7 +136,7 @@ def __eq__(self, other):
             return False
 
 
-class DifferentialOperator(object):
+class DifferentialOperator(Basic):
```

Comment:
Is it necessary to make this derive from `Basic`?

Reply:
It *could* be possible to derive this from Basic (with some trouble) but that looks like a hack to me. A better way might be implementing this [TODO](https://github.com/sympy/sympy/blob/master/sympy/interactive/printing.py#L126).

Reply:
Thank you for the advice.
I impremented the TODO, and now DifferentialOperator derive from new empty class Printable ( just for printable_types ) instead of Basic.